debian/ola.udev

```diff
@@ -10,6 +10,9 @@ SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="0962", GROUP="plug
 # udev rules for the velleman dmx device
 SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="10cf", ATTRS{idProduct}=="8062", GROUP="plugdev"
 
+# udev rules for the DMXControl Projects e.V. Nodle U1
```

Comment:
Can you either also open a PR against https://github.com/OpenLightingProject/raspberrypi/blob/master/etc/udev/rules.d/10-local.rules or bug one of us to do it please.